lib/error.g

```diff
@@ -59,16 +59,47 @@ end);
 
 ErrorLVars := fail;
 
-BIND_GLOBAL("WHERE", function( context, depth, outercontext)
-    local   bottom,  lastcontext,  f;
+BIND_GLOBAL("PRETTY_PRINT_VARS", function(context)
+    local vars, i;
+    vars := ContentsLVars(context);
+    if IsRecord(vars) then
```

Comment:
A matter of taste, but there'd be less need for indentation if you used `if not IsRecord(vars) then return; fi;`.